preview/MsixCore/MsixCoreInstaller/FileTypeAssociation.cpp

```diff
@@ -220,44 +220,22 @@ HRESULT ConvertLogoToIcon(std::wstring logoPath, std::wstring & iconPath)
     return S_OK;
 }
 
-HRESULT FileTypeAssociation::ProcessFtaForAdd(Fta& fta)
+HRESULT FileTypeAssociation::ProcessFtaForAdd(Fta& fta, HKEY rootHive)

```

Comment:
>HKEY rootHive [](start = 56, length = 13)

this rootHive specifies where WE (the msixcore installer) are writing the keys. but regardless of where we write the keys, we want to delete from both HKCU/HKLM virtual locations

Reply:
Removed this parameter. Modified to check at both HKCU and HKLM locations in the virtual mount irrespective of where the key gets written.